src/pudl/transform/eia923.py

```diff
@@ -1223,3 +1223,46 @@ def _core_eia923__fuel_receipts_costs(
     frc_df = frc_df.dropna(subset=["energy_source_code"])
 
     return frc_df
+
+
+@asset
+def _core_eia923__cooling_system_information(
+    raw_eia923__cooling_system_information: pd.DataFrame,
+) -> pd.DataFrame:
+    """Transforms the eia923__cooling_system_information dataframe."""
+    csi_df = raw_eia923__cooling_system_information
+    # Generic cleaning
+    csi_df = csi_df.pipe(pudl.helpers.fix_eia_na).pipe(pudl.helpers.convert_to_date)
+
+    # Convert cooling_status from code to description
+    csi_df["cooling_status"] = csi_df.cooling_status.str.upper().map(
+        pudl.helpers.label_map(
+            CODE_METADATA["core_eia__codes_operational_status"]["df"],
+            from_col="code",
+            to_col="description",
+            null_value=pd.NA,
+        )
+    )
+
+    # Combine the annual rates with the monthly rates
+    cols = {
+        "withdrawal_rate_gallons_per_minute": "annual_withdrawal_rate_tenth_cubic_feet_per_second",
+        "discharge_rate_gallons_per_minute": "annual_discharge_rate_tenth_cubic_feet_per_second",
+        "consumption_rate_gallons_per_minute": "annual_consumption_rate_tenth_cubic_feet_per_second",
+    }
+    for new_col, old_col in cols.items():
+        csi_df.loc[csi_df["report_date"].dt.year.isin([2008, 2009]), new_col] = csi_df[
+            old_col
+        ]
```

Comment:
The EIA923 cooling table was reported annually in 2008 and 2009 and monthly thereafter. Because all the values in the table are either static or rates (ex: water consumption in gallons/min averaged out over either the month or the year), I think it is probably ok to keep them combined. These rates to vary by month, but if there is only one month reported for the whole year, I suppose the user can assume that it's a yearly vs. a monthly average (we can also include this in the column description). I don't think it's worth separating these years out into another table.

Reply:
I like the idea of creating an annual average column and averaging the monthly values there instead of combining the annual and the monthly. Is there anything else you're confused about here that I could clarify?